diff --git a/pping.cpp b/pping.cpp
index adc172aceb8e..ed418783a264 100644
--- a/pping.cpp
+++ b/pping.cpp
@@ -55,6 +55,19 @@
  
   ***********************************************************************/
 
+#include <string>
+#include <sstream>
+
+namespace std
+{
+    template < typename T > std::string to_string( const T& n )
+    {
+        std::ostringstream stm ;
+        stm << n ;
+        return stm.str() ;
+    }
+}
+
 #include <getopt.h>
 #include <netinet/in.h>
 #include <netinet/ip.h>
