diff --git a/CMakeLists.txt b/CMakeLists.txt
index 91237edf4f2b..f184b0800665 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,8 +1,8 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.1)
 PROJECT(libtins)
 
-OPTION(LIBTINS_BUILD_EXAMPLES "Build examples" ON)
-OPTION(LIBTINS_BUILD_TESTS "Build tests" ON)
+OPTION(LIBTINS_BUILD_EXAMPLES "Build examples")
+OPTION(LIBTINS_BUILD_TESTS "Build tests")
 
 # Compile in release mode by default
 IF(NOT CMAKE_BUILD_TYPE)
